dora/core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java

```diff
@@ -1,1111 +0,0 @@
-
-import static alluxio.stress.rpc.TierAlias.MEM;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertThrows;
-import static org.junit.Assert.assertTrue;
-
-import alluxio.Constants;
-import alluxio.client.block.options.GetWorkerReportOptions;
-import alluxio.clock.ManualClock;
-import alluxio.conf.Configuration;
-import alluxio.conf.PropertyKey;
-import alluxio.exception.BlockInfoException;
-import alluxio.exception.ExceptionMessage;
-import alluxio.exception.status.NotFoundException;
-import alluxio.grpc.BuildVersion;
-import alluxio.grpc.Command;
-import alluxio.grpc.CommandType;
-import alluxio.grpc.ConfigProperty;
-import alluxio.grpc.DecommissionWorkerPOptions;
-import alluxio.grpc.RegisterWorkerPOptions;
-import alluxio.grpc.RegisterWorkerPRequest;
-import alluxio.grpc.RegisterWorkerPResponse;
-import alluxio.grpc.StorageList;
-import alluxio.grpc.WorkerLostStorageInfo;
-import alluxio.heartbeat.HeartbeatContext;
-import alluxio.heartbeat.HeartbeatScheduler;
-import alluxio.heartbeat.ManuallyScheduleHeartbeat;
-import alluxio.master.AlwaysPrimaryPrimarySelector;
-import alluxio.master.CoreMasterContext;
-import alluxio.master.MasterRegistry;
-import alluxio.master.MasterTestUtils;
-import alluxio.master.WorkerState;
-import alluxio.master.block.meta.MasterWorkerInfo;
-import alluxio.master.journal.JournalSystem;
-import alluxio.master.journal.noop.NoopJournalSystem;
-import alluxio.master.metrics.MetricsMaster;
-import alluxio.master.metrics.MetricsMasterFactory;
-import alluxio.metrics.Metric;
-import alluxio.proto.meta.Block;
-import alluxio.util.ThreadFactoryUtils;
-import alluxio.util.executor.ExecutorServiceFactories;
-import alluxio.wire.BlockInfo;
-import alluxio.wire.BlockLocation;
-import alluxio.wire.WorkerInfo;
-import alluxio.wire.WorkerNetAddress;
-import alluxio.worker.block.BlockStoreLocation;
-import alluxio.worker.block.RegisterStreamer;
-
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.Iterables;
-import com.google.common.collect.Lists;
-import io.grpc.stub.StreamObserver;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.ClassRule;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.ExpectedException;
-import org.junit.rules.TemporaryFolder;
-
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Queue;
-import java.util.concurrent.ConcurrentLinkedQueue;
-import java.util.concurrent.CyclicBarrier;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.atomic.AtomicInteger;
-import java.util.stream.Collectors;
-
-/**
- * Unit tests for {@link BlockMaster}.
- */
-public class BlockMasterTest {
-  public static final long CAPACITY = 20L * 1024 * 1024 * 1024; // 20GB
```

Comment:
deleting this test belongs to a separate PR?

Reply:
good call. removed this deletion from this PR